devtools/maven/pom.xml

```diff
@@ -191,6 +191,34 @@
             </resource>
         </resources>
         <plugins>
+            <!-- Jansi is shaded to workaround a classloading issue on Windows, see https://github.com/quarkusio/quarkus/issues/19673 -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <artifactSet>
+                                <includes>
+                                    <include>org.fusesource.jansi:jansi</include>
```

Comment:
Will this work with Maven 3.8.2?

Reply:
@knutwannheden just confirmed to me it works for both 3.8.1 and 3.8.2.
This change actually reduces the number of Jansi versions on the classpath by removing JLine which shades a Jansi version. With change we shade the Jansi version which is already on the classpath. Why exactly we have to do this to make it work is mystery to me though.